samples/getting-started/simulation/LargeSimulation.ipynb

```diff
@@ -0,0 +1,2459 @@
+       "                                    var num = 24.99999999999999;\r\n",
+       "                                    num = num.toFixed(4);\r\n",
+       "                                    var num_string = num + \"%\";\r\n",
+       "                                     document.getElementById(\"round-0bae5e0d-190a-4f2a-9e0c-bd0913b2d2d6\").innerHTML = num_string;\r\n",
+       "                                    </script> </p>\r\n",
+       "                                    </td>\r\n",
+       "                                </td>\r\n",
+       "                            \r\n",
+       "                                \r\n",
+       "                                <td style=\"transform: rotate(0deg);\r\n",
+       "                   text-align: center;\">\r\n",
+       "                                 ↑\r\n",
+       "                                </td>\r\n",
+       "                            \r\n",
+       "                            </tr>\r\n",
+       "                        \r\n",
+       "                        </tbody>\r\n",
+       "                    </table>"
+      ],
+      "text/plain": [
+       "|000000⟩\t0.4999999999999999 + 0𝑖\n",
+       "|000111⟩\t0.4999999999999999 + 0𝑖\n",
+       "|111000⟩\t0.4999999999999999 + 0𝑖\n",
+       "|111111⟩\t0.4999999999999999 + 0𝑖"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Collapse and show final state:\n",
+      "[One,One,One,One,One,One]\n",
+      "Cleaning up...\n",
+      "Done!\n"
+     ]
+    },
+    {
+     "data": {
+      "application/x-qsharp-data": "{\"@type\":\"tuple\"}",
+      "text/plain": [
+       "()"
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%simulate_sparse PrepareAndMeasureGhzState nQubits=6 nClusters=2"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "uniform-lesson",
+   "metadata": {},
+   "source": [
+    "From the histogram above you can see that the sparse simulator only stores non-zero amplitudes, as opposed to the full-state simulator (although the visualization functionality allows truncating out amplitudes that are close to zero when printing from the full-state simulator as well).\n",
+    "\n",
+    "By increasing the `nQubits` parameter you can see that the sparse simulator still provides the result quickly enough, even for qubit registers large enough that the full-state simulator slows down.\n",
+    "\n",
+    "You can increase the number of qubits up to 1,024. That is the current hardcoded limit in the sparse simulator."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ignored-sapphire",
+   "metadata": {},
+   "source": [
+    "Now you see that the sparse simulator enables users to explore larger applications than what can be represented using full-state simulator alone."
```

Comment:
Are there any CTAs we want to provide for users to continue through to?

Reply:
Adding:
## See Also
* [Quantum simulators](https://docs.microsoft.com/en-us/azure/quantum/user-guide/machines/)
* [Q# API reference](https://docs.microsoft.com/qsharp/api/qsharp/)

I also plan to add a link to [Mathias' sample](https://github.com/microsoft/Quantum/pull/619) of integer factorization with SS, once that sample is merged.
Any other good links you have in mind?